### PR TITLE
Add Fuel network

### PR DESCRIPTION
**Pull request by adlerjohn**
Implementation: https://github.com/FuelLabs/forc-wallet (specifically https://github.com/FuelLabs/forc-wallet/blob/cce7d47847fd5ba8849a6206111ea5ef9c70b735/src/account.rs#L33)

### CODE REVIEW

**Comment by prusnak** [comment]
BIP44 values need to be <= 0x8000000

File: slip-0044.md [hunk 1]
```diff
@@ -1265,6 +1265,7 @@ Coin type | Path component (`coin_type'`) | Symbol | Coin
 99999998 | 0x85f5e0fe | FLUID  | [Fluid Chains](https://www.fluidchains.com)
 99999999 | 0x85f5e0ff | QKC    | [QuarkChain](https://www.quarkchain.io)
 608589380 | 0xa4465644 | FVDC | [ForumCoin](https://fvd.nl)
+1179993420 | 0x4655454c |        | [Fuel](https://www.fuel.network)
```
Comment (prusnak) [inline slip-0044.md:1268]: Hex value should be `0xc655454c`

Comment (adlerjohn) [reply slip-0044.md]: Ah right haha I copy-pasted the value without the offset by accident, good catch!

Comment (adlerjohn) [reply slip-0044.md]: Fixed in 741d8fe

Comment (butogon) [reply slip-0044.md]: thank you

**Pull request merged**